src/shared/behaviors/domwatchable.html

```diff
@@ -0,0 +1,33 @@
+<!--
```

Comment:
this seems like an exact duplicate of dommutable only with less safety around node removal